### PR TITLE
SEXP library as git submodule

### PR DESCRIPTION
**Pull request by maxirmx**

### CODE REVIEW

File: .github/workflows/debian.yml [hunk 1]
```diff
@@ -92,9 +92,20 @@ jobs:
...
 #                                      [https://github.com/actions/runner/issues/1011]
-      - name: Checkout
+      - name: Checkout on x64
+        if: arch == 'x64'
+        uses: actions/checkout@v3
+        with:
+          submodules: true
+
+      - name: Checkout on ia64
+        if: arch == 'ia32'
```
Comment (ronaldtse) [inline .github/workflows/debian.yml:102]: @maxirmx it says "Checkout on ia64" but the condition is `ia32`. Is this correct?

Comment (maxirmx) [reply .github/workflows/debian.yml:102]: It is wrong, thank you, @ronaldtse

**Comment by maxirmx** [comment]
rnp now uses sexp library as git submodule
For build procedures it looks like sexp source is just a subfolder in src.  
We can easily change git submodule to plain subfolder if necessarily.

IMPORTANT NOTE    When release tar is created we shall check that sexp sources are inside it.

**Review by ronaldtse — APPROVED** [review]

**Review by ni4 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**